addons/arsenal/functions/fnc_fillRightPanel.sqf

```diff
@@ -273,9 +273,20 @@ switch (_ctrlIDC) do {
     };
 
     case IDC_buttonMisc : {
-        {
-            ["CfgWeapons", _x, false]  call _fnc_fill_right_Container;
-        } foreach (GVAR(virtualItems) select 17);
+        // hide custom button items
+        if (!isNil QGVAR(customRightPanelButtons)) then {
+            private _blockItems = [];
+            {
+                if (!isNil "_x") then {
+                    _blockItems append (_x select 0);
+                };
+            } forEach GVAR(customRightPanelButtons);
+            
+            {
```

Comment:
move forEach loop out of isNil scope so it always runs

Reply:
fixed, good spot